_articles/desktop-quickstart.md

```diff
@@ -24,19 +24,19 @@ If you ordered a dedicated graphics card, plug your monitor into one of the hori
 
 ### Connect Keyboard and Mouse
 
-Keyboards can connect via USB ports or wirelessly. If your keyboard/mouse connects wirelessly, you should have a small USB device, called a receiver, that connects to the desktop and communicates with your keyboard and mouse via radio frequency. This should be included when you purchase a wireless keyboard or mouse. You can connect to the front USB ports or the back, depending on where your cords will extend/fit.
+Keyboards and mice can connect via USB ports or wirelessly. Bluetooth devices can only be paired and used after new user setup. If your keyboard/mouse connects wirelessly, you should have a small USB device, called a receiver, that connects to the desktop and communicates with your keyboard and mouse via radio frequency. This should be included when you purchase a wireless keyboard or mouse. 
 
 ### Connect Power Supply
 
 The power supply is located on the bottom of the back of the desktop. Use the cord provided to connect the computer to the nearest outlet.
 
 ### Power On
 
-Press the circular button on the front of the machine after all components are plugged in. The power button is the largest button on the front of the machine.
+Press the circular button on the front of the machine after all components are plugged in. The power button is the largest button on the front of the machine. Check that the power supply is turned on at the power cable if the power button does not light up when pressed.
```

Comment:
I would probably rephrase: `If the power button does not light up when pressed, check that the power supply is turned on at the switch next to the power cable.`

Reply:
Works either way though.